docs/_documentations/codewind-vs-code-quick-guide.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: guide
+title: Codewind in VS Code
+categories: guides 
+guide_description: Take advantage of Codewind's tools to help build high quality cloud native applications regardless of which IDE or language you use.
```

Comment:
Both title: and guide_description need quotes, for example:

title: "Codewind in VS Code"
guide_description: "Take advantage of Codewind's tools to help build high quality cloud native applications regardless of which IDE or language you use."

...otherwise good to go.